# generated by patch-package 6.4.7 on 2021-05-29 12:44:39
#
# command:
#   npx patch-package @types/react-dom
#
# declared package:
#   @types/react-dom: 17.0.5
#
diff --git a/node_modules/@types/react-dom/experimental.d.ts b/node_modules/@types/react-dom/experimental.d.ts
index 9fe5335..4cf0515 100644
--- a/node_modules/@types/react-dom/experimental.d.ts
+++ b/node_modules/@types/react-dom/experimental.d.ts
@@ -1,3 +1,4 @@
+// Upstream: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/53487
 /**
  * These are types for things that are present in the `experimental` builds of React but not yet
  * on a stable build.
@@ -57,7 +58,7 @@ declare module '.' {
      *
      * @see https://reactjs.org/docs/concurrent-mode-reference.html#createroot
      */
-    function unstable_createRoot(container: Element | Document | DocumentFragment | Comment, options?: RootOptions): Root;
+    function createRoot(container: Element | Document | DocumentFragment | Comment, options?: RootOptions): Root;
 
     function unstable_flushControlled(callback: () => void): void;
 
